examples/fullasync/__main__.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+import os
+
+import uvloop
+from aiofile import AIOFile, Reader
+from roll import Roll
+from roll.extensions import cors, igniter, logger, simple_server, traceback
+
+asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
+
+app = Roll()
+cors(app)
+logger(app)
+traceback(app)
+
+
+@app.route("/fullasync", methods=["POST"], lazy_body=True)
+async def fullasync(request, response):
+    response.body = (chunk async for chunk in request)
```

Comment:
`response.body = request` would probably be shorter but I want to be sure of what is going on here.